mne/viz/_3d.py

```diff
@@ -2489,6 +2494,17 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
             dipoles, trans=trans, subject=subject, subjects_dir=subjects_dir,
             coord_frame=coord_frame, idx=idx, show_all=show_all,
             ax=ax, block=block, show=show)
+    elif mode in ['sphere', 'cone']:
+        from .backends.renderer import _Renderer
+        renderer = _Renderer(fig=fig, size=(600, 600))
+
+        pos, ori, _, _ = _get_dipole_loc(dipoles, trans, subject,
+                                         subjects_dir=subjects_dir,
+                                         coord_frame=coord_frame)
+
+        renderer.sphere(center=np.c_[pos[:, 0], pos[:, 1], pos[:, 2]],
+                        color=color, scale=1E-3)
```

Comment:
dipoles are not expect to be sphere. They have a position and an orientation. It's better to use Arrows like:

https://mne-tools.github.io/dev/auto_tutorials/sample-datasets/plot_brainstorm_phantom_elekta.html?highlight=plot_brainstorm_phantom_elekta

Reply:
Some tutorials were more interested in the locations only actually.

Reply:
Like this one: `plot_phantom_4DBTi.py`

Reply:
And the first plot of `plot_dipole_orientations.py` I think.